### ISSUE TITLE
Internal server error when loaning out items that are in maintenance

### ISSUE THREAD

**Issue by phinze** [chicago-tool-library/circulate#1247]
**Describe the bug**

When trying to hold or lend items in maintenance, the app generates an error:

```
undefined method `persisted?' for false:FalseClass

      if @loan.persisted?
              ^^^^^^^^^^^
```

Coming from `app/controllers/admin/loans_controller.rb:21 create`

**To Reproduce**
1. As an admin, attempt to loan an item that is in maintenance

**Expected behavior**
1. Items in maintenance should produce a validation error when attempting to hold or loan them, not an error page. 

**Additional context**
* Appsignal page for the error: https://appsignal.com/chicago-tool-library/sites/60596f9214ad662e17191689/exceptions/incidents/225

**Comment by phinze** [comment]
I think to fix this we should make the [create_loan method](https://github.com/chicago-tool-library/circulate/blob/main/app/controllers/concerns/lending.rb#L7-L14) always return the `Loan` object even if it fails to save. That way the LoansController can properly check `persisted?` and pass along the error messages to the UI.

**Comment by jpeace-pro** [comment]
Resolved in [PR #1263](https://github.com/chicago-tool-library/circulate/pull/1263)!
